package.json

```diff
@@ -41,8 +41,58 @@
     "x-default-browser": "^0.5.2"
   },
   "devDependencies": {
+    "@storybook/react": "^6.5.16",
+    "@testing-library/jest-dom": "^6.4.6",
+    "@testing-library/react": "^15.0.7",
+    "@testing-library/user-event": "^14.5.2",
+    "@types/jest": "^29.5.12",
+    "@types/node": "^20.14.9",
+    "@types/react": "^18.3.3",
+    "@types/react-dom": "^18.3.0",
+    "@types/react-highlight": "^0.12.8",
+    "@types/react-router-dom": "^5.1.1",
+    "@types/react-transition-group": "^4.4.10",
+    "@types/styled-components": "^5.1.34",
+    "@types/wicg-file-system-access": "^2023.10.5",
+    "jest": "^29.7.0",
```

Comment:
Initially, I wanted to keep `jest` in `packages/build`, but I didn't think about two things:
1. I need `@types/jest` globally (so TS can pick it); this suggests that `jest` should be kept in the same `package.json`.
2. Adding a script to `packages/build` and then calling it from the root,  runs only tests from `packages/build`. I'd have to change the current working directory probably, but I guess it's not worth it.

So, instead of keeping tooling in `packages/build`, it will be more a place to for the tooling configuration (there are still many plugins/configs there).